### PR TITLE
Added strikethrough for git basics

### PR DESCRIPTION
**Pull request by jerome3o**
Making a test merge request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +12,23 @@ git commit
 git push
 ``` 
 
+Or you can make a branch
```
Comment (jerome3o) [inline README.md:15]: please add another section here

Comment (jerome3o) [reply README.md:15]: ah nah dog that aint me

Comment (Samueljbk) [reply README.md:15]: Not about that life soz

File: README.md [hunk 2]
```diff
@@ -12,10 +12,23 @@ git commit
...
+
+```sh
+git checkout -b feature/feature-description
+git push --upstream origin feature/feature-description
+# make some changes
+git add .
+git commit -m "implemented feature XYZ"
+git push
+```
+
```
Comment (Samueljbk) [inline README.md:25]: Great moves Jerome, keep it up, proud of you!

**Review by Samueljbk — APPROVED** [review]
Very pog

File: README.md (hunk 2 above)
Comment (jerome3o) [reply README.md:25]: ty fam

**Pull request merged**